packages/types/src/status.ts

```diff
@@ -0,0 +1,44 @@
+/** The status of an event. */
+export enum Status {
```

Comment:
we can also consider our Amplitude-specific `silenced` status 400 here - specific to when the data will never be accepted due to [rare, device-specific cases](https://docs.google.com/document/d/1-UllDpUi9z6tsS8G6u1PR-9Y2Wi5NI4eWbQ_EmVbDXs/edit#)

Reply:
Bear in mind 400 on its own doesn't mean that I think. At least on the more recent iterations of the APIs there should be an explicit message indicating that.

400 can also be returned for reasons like malformed event payloads.